docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,244 @@
+Both of these offsets will prevent frontrunning (as the profit was already earned and was not distributed yet).
+
+Instead of locking the profit, and to be more capital efficient (and avoid complex asset management), the vault issues shares that holds itself. Those shares are gradually burned over time (increasing price-per-share). So instead of releasing assets, the vault will gradually burn shares.
+
+## Vault Management
+Vault management is split into two fields: strategy management and debt management.
+
+### Strategy Management
+Callers take this responsibility with the `STRATEGY_MANAGER` role.
+
+A vault can have strategies added, removed, and migrated.
+
+Added strategies will be eligible to receive funds from the vault when `max_debt` is set to > 0.
+
+Revoked strategies will return all debt and be ineligible to receive more. This happens when the strategy's `current_debt` is 0.
+
+Strategy migration is the process of replacing an existing strategy with a new one, which will inherit all of the old strategy's parameters, including its debt.
+
+### Debt Management
+Callers take this responsibility with the `DEBT_MANAGER` role.
+
+#### Setting minimum idle funds
+The `DEBT_MANAGER` can specify how much funds the vault should try to have reserved to serve withdrawal requests.
+
+These funds will remain in the vault unless requested by a Depositor.
+
+#### Setting maximum debt for a specific strategy
+The maximum amount of tokens the vault will allow a strategy to owe at any moment.
+
+Stored in `strategies[strategy].max_debt`.
+
+When a debt rebalance is triggered, the Vault will cap the new target debt to this number (max_debt).
+
+#### Rebalance Debt
+The vault sends and receives funds to/from strategies. The function updateDebt(strategy, target_debt) will set the current_debt of the strategy to target_debt (if possible).
+
+If the strategy currently has less debt than the target_debt, the vault will send funds to it.
+
+The vault checks that the `minimumTotalIdle` parameter is respected (i.e. there's at least a certain amount of funds in the vault).
+
+The vault will request back funds if the strategy has more debt than the `max_debt`. These funds may be locked in the strategy, which will result in the strategy returning fewer funds than requested by the vault. 
+
+## Roles
+Vault functions that are permissioned will be callable by accounts with specific roles. 
+
+These are: 
+- `STRATEGY_MANAGER:` role for accounts that can add, remove or migrate strategies.
+- `DEBT_MANAGER:` role for accounts that can rebalance and manage debt related params.
+- `EMERGENCY_MANAGER:` role for accounts that can activate the shutdown mode.
+- `ACCOUNTING_MANAGER:` role for accounts that can process reports (save strategy p&l) and change accountant parameters.
+
+Every role can be filled by an EOA, multisig, or other smart contracts. Each role can be filled by several accounts.
+
+The account that manages roles is a single account, set in `role_manager`.
```

Comment:
why is `role_manager` not capped while `STRATEGY_MANAGER` is capped?

Reply:
I see that makes sense not sure if it needs to be added in this document though

Reply:
Yes that makes sense thanks, but I dont think this should be added in this document